### ISSUE TITLE
COAM - COAM Grid

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-obps#301]
#### Description:

As an external user, I can view a grid of compliance obligations for each of my operations

#### Acceptance Criteria:

- [ ] Given I have navigated to the Compliance Obligation Grid, I see a grid of each of my operations and their compliance obligation amounts.
    - Amount is the emissions total pulled from reporting, displayed in tCO2e. If nothing is owed, show the amount of credits being earned: $0 owed. [amount] credits earned
- [ ] Given I am on the grid, I can see a dropdown with a list of reporting years, and when I select a year, the grid will show the compliance obligations for each operation for that selected year 
- [ ] Given I am on the Compliance Obligation Grid, I can see my COAM balance being automatically adjusted as my payments are being made.
    - If I edit and re-submit a report, it will show the new amount of tCO2e owing or earned
    - 
#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking
